fx2eeprom.c

```diff
@@ -60,19 +60,23 @@ int main(int argc, char *argv[]){
 	int address=0;
 	int mode;
 
-	if (argc!=5){
+	if (argc<4){
 		use(argv[0]);
 		return -1;
 	}
 
+	if (argc==5)
```

Comment:
Why do you want to provide a default memory size? Doesnt it depend on the chip model?

Reply:
The memory size is most important for read operation, if I e.g. want to get the 1st 8 bytes from a C0 section. But also writing can be sized, e.g. `fx2eeprom w $VID $PID 8192 </dev/zero` to erase the EEPROM. This is also the reason I invented the EEPROM memory address as 5th optional parameter.